sonar-project.properties

```diff
@@ -11,7 +11,7 @@ sonar.links.homepage=https://github.com/CDOT-CV/Work_Zone
 sonar.links.scm=https://github.com/CDOT-CV/Work_Zone
 sonar.links.issue=https://github.com/CDOT-CV/Work_Zone/issues
 sonar.python.coverage.reportPaths=*coverage*.xml
-sonar.exclusions=*coverage*.xml,test/*,translator/GCP_cloud_function/**
+sonar.exclusions=*coverage*.xml,test/*,translator/GCP_cloud_function/cloud_function/icone_translator.py
```

Comment:
Is the `icone_translator.py` under "cloud_function" directory the same as the one under "source_code" directory? If so, let's just have the single code file and import it to the cloud function. Then we can remove the one under the "cloud_function" directory and not have to do this odd exclusion.

Reply:
Removed icone_translator.py from "GCP_cloud_function" directory and mader recomended changes and pushed it to dev.

Reply:
Let's remove the icone_translator.py sonar exclusion as well. The file no longer exists, so no need to exclude it.